build/dev-env.sh

```diff
@@ -47,21 +47,21 @@ fi
 
 HELM_VERSION=$(helm version 2>&1 | grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+') || true
 if [[ ${HELM_VERSION} < "v3.0.0" ]]; then
```

Comment:
If we force using Helm v3.9.0+, then we need change this condition as well

Reply:
I can do this change. But some people may not have updated helm. So I left it at that. This way if it fails, then suggestion is to install v3.9.0 else if it passes, they can continue to use whatever version they have and upgrade helm as per their timeline. I will wait for your comment and change or keep it, based on your comment.

Reply:
@tao12345666333, on second thoughts, I changed as per suggestion